pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -220,7 +220,7 @@ type KubeAPIServerInformers struct {
 	InternalKubernetesInformers        kinternalinformers.SharedInformerFactory
 	KubernetesInformers                kexternalinformers.SharedInformerFactory
 	OpenshiftOAuthInformers            oauthinformer.SharedInformerFactory
-	InternalOpenshiftImageInformers    imageinformer.SharedInformerFactory
+	InternalOpenshiftImageInformers    imagev1informer.SharedInformerFactory
```

Comment:
should this field be renamed if it's not an internal informer anymore?

Reply:
Thanks, done